docs/runtime.md

```diff
@@ -0,0 +1,138 @@
+# RedisGears Functions Runtime
+
+Python RedisGears functions are run using an embedded Python interpreter. Each function uses a separate sub-interpreter. All functions share the same environment and dependencies. The environment is imported with several defaults.
+
+The following sections describe the runtime environment.
+
+## Python Interpreter
+RedisGears embeds a Python version 3.7.2+ interpreter.
+
+All functions use that interpreter. Each function has its sub-interpreter that isolates its execution context from other functions. This means each function has its globals, function and class names.
+
+!!! info "Further reference"
+    For more information refer to:
+
+      * [Sub-interpreters design](subinterpreters.md)
+
+## Environment
+The interpreter's environment can be extended with any dependent package that can later be imported and used by functions in their respective sub-interpreters.
+
+!!! info "Further reference"
+    For more information about installing dependencies refer to:
+
+      * [The `REQUIREMENTS` subcommand of `RG.PYEXECUTE`](commands.md#rgpyexecute)
```

Comment:
It more of an argument then a subcommand ..

Reply:
Fixed via 079f280